community_images/common/orchestrator/k8s_setup.py

```diff
@@ -186,6 +196,28 @@ def create_helm_chart(self, cmd):
         logging.info(f"cmd: {cmd}")
         Utils.run_cmd(cmd.split())
 
+    def run_health_check(self):
+        """ Run health check """
+        health_check_script = self.runtime_props.get("health_check_script", "")
+        if health_check_script is None:
+            return
+
+        logging.info(f"running health check script: {health_check_script}")
+        timeout = self.runtime_props.get("health_check_timeout", 300)
+        # run the health check script
+        script_path = f"{self.image_script_dir}/{health_check_script}"
+        cmd = f"bash {script_path} {self.namespace_name} {self.release_name}"
+        try:
+            output = subprocess.check_output(
```

Comment:
Why are we not using Utils.run_cmd ?

Reply:
we can use, have added timeout param to run_cmd function